src/targets/__tests__/maven.test.ts

```diff
@@ -143,9 +151,11 @@ describe('publish', () => {
     mvnTarget.artifactProvider.downloadArtifact = jest
       .fn()
       .mockResolvedValueOnce('artifact/download/path');
+    mvnTarget.isBomFile = jest.fn().mockResolvedValueOnce(undefined);
```

Comment:
`isBomFile` returns `Promise<boolean>`
```suggestion
    mvnTarget.isBomFile = jest.fn().mockResolvedValueOnce(false);
```